src/traces/contour/make_color_map.js

```diff
@@ -0,0 +1,78 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+
+'use strict';
+
+var d3 = require('d3');
+
+var getColorscale = require('../../components/colorscale/get_scale');
+
+
+module.exports = function makeColorMap(trace) {
+    var contours = trace.contours,
+        start = contours.start,
+        end = contours.end,
+        cs = contours.size || 1,
+        nc = Math.floor((end + cs / 10 - start) / cs) + 1,
```

Comment:
Can we use more verbose names? `cs` I can understand as `contour size` but I have no idea what `nc` is.